sbol2/property.py

```diff
@@ -729,7 +740,7 @@ def clear(self):
                 object_store = self._sbol_owner.owned_objects[self._rdf_type]
                 for obj in object_store:
                     if obj.is_top_level() and obj.doc is not None:
-                        obj.doc.SBOLObjects.remove(obj.identity)
+                        obj.doc.SBOLObjects.remove(rdflib.URIRef(obj.identity))
```

Comment:
Do we want to make the `remove` method accept both string and URIRef types?  Maybe this should be a new issue

Reply:
I changed Document.SBOLObjects to be a `URIDict`, which will take either strings or URIRefs as keys. And I changed the line above to remove the cast and just use `obj.identity`

Also, that method does not get invoked by anything in the unit tests.